app/response/views.py

```diff
@@ -170,6 +171,39 @@ def response_link(request_id):
     return redirect(url_for('request.view', request_id=request_id))
 
 
+@response.route('/instruction/<request_id>', methods=['POST'])
+def response_instructions(request_id):
+    """
+    Instruction response endpoint that takes in from the frontend, the content of a instruction for a specific request.
+    Check if required data from form is retrieved.
+    Call add_instruction to process the extension form data.
+
+    :param request_id: FOIL request ID for the specific note.
+    :return: Flash error message if required form data is missing.
+             Redirect to view request page as of right now (IN DEVELOPMENT) upon endpoint function completion.
+    """
+    instruction_data = flask_request.form
+
+    required_fields = ['content',
+                       'email-instruction-summary',
+                       'privacy']
+
+    # TODO: Get copy from business, insert sentry issue key in message
```

Comment:
I believe it's to get the correct error message content from the business owners.